packages/react-core/src/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -53,51 +61,68 @@ import { MenuToggle, Badge } from '@patternfly/react-core';
 
 ```
 
-### Primary
+### With icons
+
+Add recognizable icons to a menu toggle using the `icon` property.
 
 ```ts
 import React from 'react';
-import { MenuToggle } from '@patternfly/react-core';
+import { MenuToggle, Avatar } from '@patternfly/react-core';
 import CogIcon from '@patternfly/react-icons/dist/esm/icons/cog-icon';
 
 <React.Fragment>
-  <MenuToggle variant="primary">Collapsed</MenuToggle>{' '}
-  <MenuToggle variant="primary" icon={<CogIcon />}>
-    Icon
-  </MenuToggle>{' '}
-  <MenuToggle variant="primary" isExpanded>
-    Expanded
-  </MenuToggle>{' '}
-  <MenuToggle variant="primary" isDisabled>
-    Disabled
-  </MenuToggle>
+  <MenuToggle icon={<CogIcon />} variant="secondary">Icon</MenuToggle>{' '}
+  <MenuToggle icon={<CogIcon />} variant="secondary" isDisabled>Icon</MenuToggle>
 </React.Fragment>
+```
+
+### With avatar and text
 
+To add an avatar to a toggle, pass an `<Avatar>` component into the toggle's `icon` property. 
+
+This can be used alongside a text label to display a user's login profile, as shown in the following example.
+
+```ts
+import React from 'react';
+import { MenuToggle, Avatar } from '@patternfly/react-core';
+import imgAvatar from '@patternfly/react-core/src/components/Avatar/examples/avatarImg.svg';
+
+<React.Fragment>
+  <MenuToggle icon={<Avatar src={imgAvatar} alt="avatar" />}>Ned Username</MenuToggle>{' '}
+  <MenuToggle icon={<Avatar src={imgAvatar} alt="avatar" />} isExpanded>Ned Username</MenuToggle>{' '}
+  <MenuToggle icon={<Avatar src={imgAvatar} alt="avatar" />} isDisabled>Ned Username</MenuToggle>
+</React.Fragment>
 ```
 
-### Secondary
+### Variant styles
+
+Variant styling can be applied to menu toggles. In the following example, the toggle uses primary styling by passing `variant="primary"` into the `<MenuToggle>` component. Additional variant options include “default”, “plain”, “plainText”, “secondary”, and “typeahead”.
 
 ```ts
 import React from 'react';
 import { MenuToggle } from '@patternfly/react-core';
 import CogIcon from '@patternfly/react-icons/dist/esm/icons/cog-icon';
 
 <React.Fragment>
-  <MenuToggle variant="secondary">Collapsed</MenuToggle>{' '}
-    <MenuToggle variant="secondary" icon={<CogIcon />}>
+  <MenuToggle variant="primary">Collapsed</MenuToggle>{' '}
+  <MenuToggle variant="primary" icon={<CogIcon />}>
     Icon
   </MenuToggle>{' '}
-  <MenuToggle variant="secondary" isExpanded>
+  <MenuToggle variant="primary" isExpanded>
     Expanded
   </MenuToggle>{' '}
-  <MenuToggle variant="secondary" isDisabled>
+  <MenuToggle variant="primary" isDisabled>
     Disabled
   </MenuToggle>
 </React.Fragment>
 
 ```
 
-### Plain
+### Plain toggle with icon
+
+To apply plain styling to a menu toggle, pass in `variant="plain"`. You may pass in an `icon` to serve at the menu toggle. The following example passes in an `<EllipsisVIcon />` icon. 
+
+To include an aria label, use the `aria-label` property. 
```

Comment:
```suggestion
To apply plain styling to a menu toggle, pass in `variant="plain"`. This will remove the default bottom border and caret. You may pass in an `icon` to serve as the menu toggle. The following example passes in an `<EllipsisVIcon />` icon. 

If the toggle does not have any visible text content, use the `aria-label` property to provide an accessible name. 
```
Just a small tweak to the description to note what `plain` does (though since you mention what `plainText` does below it may be redundant). Depending on whether the intention is for `plain` to only be used for toggles with an icon only, it may be worth mentioning that as well ("To apply plain styling to a menu toggle that only contains an icon...") cc @mcarrano